src/main/java/org/breedinginsight/api/model/v1/response/DataResponse.java

```diff
@@ -29,5 +30,6 @@
 @NoArgsConstructor
 @AllArgsConstructor
 public class DataResponse<T> {
+    @JsonInclude()
```

Comment:
After looking up @JsonInclude, I see that it defaults to `JsonInclude.Include.ALWAYS`. For me personally seeing `JsonInclude.Include.ALWAYS` would make it more clear but IntelliJ doesn't like that and says it's redundant. Didn't know if anyone else had any thoughts on this? I'm ok with it either way just didn't know if we had a preferred style we should follow.

Reply:
I don't see a problem with leaving it as `@JsonInclude`

Reply:
I was using `JsonInclude.Include.ALWAYS` at first in the other models, but I think I lean toward not including it now. Just having the `@JsonInclude` seems explicit enough to me. But, thats a good point on consistency, because some of our models will still have the `JsonInclude.Include.ALWAYS` so we should update those at some point.